cmd/rook/ceph/mds.go

```diff
@@ -82,3 +75,14 @@ func startMDS(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func extractMdsID(mdsName string) string {
+	prefix := "rook-ceph-mds-"
```

Comment:
Could the const for the `rook-ceph-mds` be used here? This be used here https://github.com/rook/rook/blob/master/pkg/operator/ceph/file/mds.go#L39 ?